opendebates/views.py

```diff
@@ -352,6 +352,16 @@ def questions(request):
     return redirect(url + "#created=%s" % idea.id)
 
 
+@rendered_with("opendebates/changelog.html")
+def changelog(request):
+    moderated = Submission.objects.filter(
+        Q(approved=False) | Q(duplicate_of__isnull=False)
```

Comment:
I think these two sets can overlap - is that a problem?  Do we need a `.distinct()` tacked on?

(If `idea.duplicate_of and not idea.approved`, then the idea had been marked as a duplicate without being fully merged because its vote tally was too low to merit a full merge.  It looks like this option isn't being used on presidentialopenquestions.com but it's supported in the codebase.)